.github/workflows/Build.yaml

```diff
@@ -0,0 +1,146 @@
+name: Build, Test and Release
+
+on:
+    pull_request:
```

Comment:
> By default, a workflow only runs when a pull_request's activity type is opened, synchronize, or reopened. To trigger workflows for more activity types, use the types keyword.
 -- https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

Could you add explicit `types` to make sure the activity type is clear?

Reply:
Added explicit types: [opened, synchronize, reopened]